packages/feathers/lib/index.js

```diff
@@ -3,9 +3,12 @@ const Proto = require('uberproto');
 const Application = require('./application');
 const version = require('./version');
 const { ACTIVATE_HOOKS, activateHooks } = require('./hooks');
+// A base object Prototype that does not inherit from a
+// potentially polluted Object prototype
+const baseObject = Object.create(null);
 
 function createApplication () {
-  const app = {};
+  const app = Object.create(baseObject);
```

Comment:
That would not be enough to do that?

```js
const app = Object.create(null);
```

Reply:
I thought so, too but [Uberproto](https://github.com/daffl/uberproto) tries to [get the Prototype](https://github.com/daffl/uberproto/blob/master/lib/proto.js#L74) when mixing into the `app` object, gets `null` and blows up so I needed to add another one to the chain.